l10n-ms/src/main/java/com/example/l10nadmin/domain/requestForm.java

```diff
@@ -0,0 +1,20 @@
+package com.example.l10nadmin.domain;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+@Data // = @ToString, @EqualsAndHashCode, @Getter, @Setter and @RequiredArgsConstructor
+public class requestForm {
```

Comment:
```suggestion
public class RequestForm {
```

Must be in CapitalCamelCase